apps/src/aichat/views/UserChatMessageEditor.tsx

```diff
@@ -28,13 +28,7 @@ const UserChatMessageEditor: React.FunctionComponent = () => {
 
   const disabled = isWaitingForChatResponse || saveInProgress;
 
-  return (
-    <UserMessageEditor
-      onSubmit={handleSubmit}
-      disabled={disabled}
-      showSubmitLabel
-    />
-  );
+  return <UserMessageEditor onSubmit={handleSubmit} disabled={disabled} />;
```

Comment:
Forgot to hide the "Submit" button label in a previous PR (request from product/design), so including that here.